x/simulation/util.go

```diff
@@ -91,3 +76,12 @@ func PeriodicInvariants(invariants []sdk.Invariant, period int, offset int) []sd
 	}
 	return outInvariants
 }
+
+func mustMarshalJSONIndent(o interface{}) []byte {
```

Comment:
Moving `mustMarshalJSONIndent()` to `tests` would spare us from code duplication